.gitignore

```diff
@@ -102,3 +102,6 @@ scan-journal.log
 
 # connectors' cache
 *.sqlite
+
+# poetry
+*poetry.lock
```

Comment:
Include poetry.lock in all your connectors, they need to be checked in to the repo.